### DISCUSSION TITLE
build failed in m1 Mac for version 2.6.10 and 2.7.6

### DISCUSSION THREAD

**Discussion by GeorgeMpouk** [rbenv/ruby-build#2078] [Build failures]
Installing ruby-2.6.10...
ruby-build: using readline from homebrew

BUILD FAILED (macOS 13.0 using ruby-build 20221101)

Inspect or clean up the working tree at /var/folders/1g/vmkjmlxx7b3g1rg2dq8ffyyh0000gn/T/ruby-build.20221102014625.51809.Hlf66N
Results logged to /var/folders/1g/vmkjmlxx7b3g1rg2dq8ffyyh0000gn/T/ruby-build.20221102014625.51809.log

That was the error when I tried installing the 2.6.10 version so that I can maybe install 2.7.6 later

**Comment by mislav** [comment]
Hi, please see my answer https://github.com/rbenv/ruby-build/discussions/2076#discussioncomment-4030643

In short, if you want to share a build failure with others, make sure to include the link to an uploaded log file.
